test/test-fs-event.c

```diff
@@ -109,6 +109,12 @@ static void fail_cb(uv_fs_event_t* handle,
   ASSERT(0 && "fail_cb called");
 }
 
+static void noop_cb(uv_fs_event_t* handle,
+                    const char* path,
+                    int events,
+                    int status) {
+}
```

Comment:
MAKE_VALGRIND_HAPPY will start the loop after all handles are closed in order for the close callbacks to run. This callback should never be called anyway.